GA/post.py

```
@@ -755,51 +798,139 @@ def generate_convergence_statistics(self,
+                    f = running_metric.delta_f
+                    x = np.arange(len(f)) + 1
+                    v = [max(ideal, nadir) >1E-5 for ideal, nadir in zip(running_metric.delta_ideal, running_metric.delta_nadir)]
+
+                else:
+                    running_metric.update(e)
+                    tau = e.n_gen
+                    f = running_metric.delta_f
+                    x = np.arange(len(f)) + 1
+                    v = [max(ideal, nadir) >1E-5 for ideal, nadir in zip(running_metric.delta_ideal, running_metric.delta_nadir)]
+
+                    ax_big.plot(x, f, label=f"Final population", alpha=0.9)
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)